lib/stat_tracker.rb

```diff
@@ -60,35 +60,44 @@ def percentage_ties
   def count_of_games_by_season
     Game.count_of_games_by_season
   end
-  #ross
```

Comment:
no one needs to see my name

Reply:
![](https://media0.giphy.com/media/xULW8N9O5WD32L5052/giphy.gif?cid=5a38a5a2cb111c12cc8eaf63edcad3dd6680180b55d2aab1&rid=giphy.gif)